public/build-integrations/index.html

```diff
@@ -16,19 +16,19 @@
 
 <meta name="description" content="Build integrations with Squadcast." />
 
-<meta property="og:image" content="https://developers.squadcast.com"/>
+<meta property="og:image" content="http://localhost:1313/"/>
```

Comment:
@ShubhanjanMedhi-dev these are localhost links. Is this fine?

Reply:
this is inside public folder so its fine it will get generated again after merging, pushing the updated ones anyways